src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -86,6 +86,13 @@ public void add() {
         command = PersonUtil.getAddCommand(toAdd);
         assertCommandSuccess(command, toAdd);
 
+        /* Case: using alias to add a person without tags to a non-empty address book, command with leading spaces and trailing spaces
```

Comment:
Line is longer than 120 characters (found 134) - split this comment into 2 lines to fix code style violation